app/main/db/queries.py

```diff
@@ -70,3 +70,42 @@ def fuzzy_search(query_string):
     except exc.SQLAlchemyError as e:
         print("Failed to return results from database with error : " + str(e))
     return results
+
+
+def browse_view():
+    results = []
+
+    query = (
+        db.select(
+            Body.Name.label("TransferringBody"),
+            Series.Name.label("Series"),
+            func.max(Consignment.TransferCompleteDatetime).label(
+                "Last_Record_Transferred"
+            ),
+            func.count(func.distinct(Consignment.ConsignmentReference)).label(
+                "Consignment_in_Series"
```

Comment:
Could we make all these labels snake_case?

e.g.

```suggestion
                "consignment_in_series"
```

Reply:
variables are set with snake_case a requested